spec/list_spec.rb

```diff
@@ -249,5 +249,24 @@ module Trello
         end
       end
     end
+
+    describe '#move_to_board' do
+      let(:board) { Trello::Board.new(id: 'someid') }
+      it 'moves to a Trello::Board' do
```

Comment:
What do you think about adding a `context` into this spec to make the spec readability better?

```
context "given the destination board parameter is a Trello::Board instance" do
  it "moves to the destination board" do
    # ----
  end
end
```

And the same goes for the 2nd spec

Reply:
Feels way to verbose IMHO, but I'm more of a minispec guy. I've done it anyway, and rebased to ensure a clean history.